spec.md

```diff
@@ -3459,6 +3498,9 @@ primop_1expr     = primop_1expr_keyword , "(" , expr , ")" ;
 primop_1expr1int = primop_1expr1int_keyword , "(", expr , "," , int , ")" ;
 primop_1expr2int = primop_1expr2int_keyword , "(" , expr , "," , int , "," , int , ")" ;
 
+(* Primitive Property Operations *)
+property_primop_2expr = property_primop_2expr_keyword , "(" , property_expr , "," , property_expr ")" ;
```

Comment:
Hmm in the rendered PDF this overflows... I will try breaking this differently.

Reply:
Yes... the grammar is verbatim and you have to do some manual reflowing to get things to fit. There are other examples of manual breaking in the grammar where this happens.

Reply:
Updated slightly, this is what I went with.
<img width="1296" alt="Screen Shot 2024-02-08 at 5 19 07 PM" src="https://github.com/chipsalliance/firrtl-spec/assets/1182100/cb486d98-9c34-4577-8b81-c3f43f57f2d0">

Open to other indentation ideas.